CONTRIBUTING.md

```
@@ -50,12 +50,13 @@ git checkout -b type/краткое-описание
 ### 3. Внесение изменений
 **Для текста учебника**:
 - Размещайте изменения в соответствующих файлах в `src/chapters/`
-- Используйте семантические коммиты:
+- Используйте [семантические коммиты](https://www.conventionalcommits.org/en/v1.0.0/):
   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)